lib/iris/tests/test_nimrod.py

```diff
@@ -73,7 +73,14 @@ def test_huge_field_load(self):
             "probability_fields",
         }:
             cube = iris.load(
-                tests.get_data_path(("NIMROD", "uk2km", "cutouts", datafile,))
+                tests.get_data_path(
+                    (
+                        "NIMROD",
+                        "uk2km",
+                        "cutouts",
+                        datafile,
+                    )
+                )
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?